tests/test_config.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from module import ConfigParser
+import os
+
+
+class TestConfig:
+
+    def setUp(self):
+        ConfigParser.pypath = ".."
+        self.config = ConfigParser.ConfigParser()
+
+    def teardown(self):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=4846282)